src/components/AutoComplete/AutoComplete.tsx

```diff
@@ -92,6 +91,8 @@ function AutoComplete({
           disabled && 'opacity-30 bg-gray-100',
           className
         )}
+        min={undefined}
```

Comment:
Why you need to pass min and max on this Input? 🤔

Reply:
I already resolved it